strings/ion_laws.json

```diff
@@ -153,6 +153,7 @@
         "WHITE",
         "BROWN",
         "GREY",
+        "ERECT",
```

Comment:
Nah, we already have the adjective "HARD", no need for "ERECT".

I remember one time as AI, I got a law that "THE STATION REQUIRES FIVE HARD MIMES"  So I set the availible jobs to only mimes, and told my secborg to give mimes lapdances.

Good times.